docs/src/modules/components/ApiDocs.js

```diff
@@ -0,0 +1,391 @@
+  const { componentProps, propDescriptions } = props;
+  const t = useTranslate();
+  const userLanguage = 'en';
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th align="left">{t('name')}</th>
+          <th align="left">{t('type')}</th>
+          <th align="left">{t('default')}</th>
+          <th align="left">{t('description')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {Object.entries(componentProps).map(([propName, propData]) => {
+          const typeDescription = propData.type.description || propData.type.name;
+          const propDefault = propData.default || (propData.type.name === 'bool' && 'false');
+          return (
+            propData.description !== '@ignore' && (
+              <tr key={propName}>
+                <td align="left">
+                  <span className={clsx('prop-name', propData.required ? 'required' : null)}>
+                    {propName}
+                    {propData.required ? <sup>*</sup> : null}
+                  </span>
+                </td>
+                <td align="left">
+                  {typeDescription.length > 20 ? (
+                    <details className="prop-type">
+                      <summary>{propData.type.name}</summary>
+                      <span dangerouslySetInnerHTML={{ __html: typeDescription }} />
+                    </details>
+                  ) : (
+                    <span
+                      className="prop-type"
+                      dangerouslySetInnerHTML={{ __html: typeDescription }}
+                    />
+                  )}
+                </td>
+                <td align="left">
+                  {propDefault && <span className="prop-default">{propDefault}</span>}
+                </td>
+                <td
+                  align="left"
+                  dangerouslySetInnerHTML={{
+                    __html: propDescriptions[userLanguage][propName] || '',
+                  }}
+                />
+              </tr>
+            )
+          );
+        })}
+      </tbody>
+    </table>
+  );
+}
+
+PropsTable.propTypes = {
+  componentProps: PropTypes.object.isRequired,
+  propDescriptions: PropTypes.object.isRequired,
+};
+
+function ClassesTable(props) {
+  const { componentName, componentStyles, classDescriptions } = props;
+  const t = useTranslate();
+  const userLanguage = 'en';
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th align="left">{t('ruleName')}</th>
+          <th align="left">{t('globalClass')}</th>
+          <th align="left">{t('description')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {componentStyles.classes.map((className) => (
+          <tr key={className}>
+            <td align="left">
+              <span className="prop-name">{className}</span>
+            </td>
+            <td align="left">
+              <span className="prop-name">
+                {componentStyles.globalClasses[className] || `Mui${componentName}-${className}`}
```

Comment:
To emphasize the CSS selector

```suggestion
                .{componentStyles.globalClasses[className] || `Mui${componentName}-${className}`}
```